### DISCUSSION TITLE
v0.6.1

### DISCUSSION THREAD

**Discussion by U-C-S** [U-C-S/Hurl#40] [Announcements]
## Features Updates
- Support `DisableAcrylic`, `BackgroundRGB` options in settings (part of #39)
- Use Horizontal Scrollbars after 5 browsers in the view (#25)

## Fixes
- Fix Tray Reload Option Error
- Various unknown crashes

## Development Updates
- Update to WPFUI v1.2.3 (Also, Use WPFUI as git submodule)
- Code Refactoring and Improvements

**Full Changelog**: https://github.com/U-C-S/Hurl/compare/v0.6.0...v0.6.1

<hr /><em>This discussion was created from the release <a href='https://github.com/U-C-S/Hurl/releases/tag/v0.6.1'>v0.6.1</a>.</em>

**Comment by grentninja** [comment]
Hurl doesn't run in my Windows 10 now :/ I wonder why...

**Reply by grentninja** [reply]
I managed to fix it. Because I messed with my UserSettings.json, I added alternative launchers etc., then it did not update my UserSettings.json, I think. So when I updated to 0.6.1, it does not launch. So, I changed the UserSettings.json from 0.6 to 0.6.1 by adding the comment given to me in issue #39. It went like this: (I added the said app settings & changed the version) since it does not change it on its own.
```json
{
  "LastUpdated": "22/04/2022 2:27:26 pm",
  "Version": "0.6.1",
  "AppSettings": {
      "DisableAcrylic": true,
      "BackgroundRGB": [38, 38, 38]
   },
  "Browsers": [
    {
      "Name": "Brave",
      "ExePath": "C:\\Program Files\\BraveSoftware\\Brave-Browser\\Application\\brave.exe",
      "LaunchArgs": null,
      "Hidden": false,
      "AlternateLaunches": [
      {
    "ItemName": "Incognito",
    "LaunchArgs": "-incognito"
      }
    ]
    },
    {
      "Name": "Firefox",
      "ExePath": "C:\\Program Files\\Mozilla Firefox\\firefox.exe",
      "LaunchArgs": null,
      "Hidden": false,
      "AlternateLaunches": [
      {
    "ItemName": "Incognito",
    "LaunchArgs": "-private-window"
      }
    ]
    },
    {
      "Name": "Microsoft Edge",
      "ExePath": "C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe",
      "LaunchArgs": null,
      "Hidden": false,
      "AlternateLaunches": [
      {
    "ItemName": "Incognito",
    "LaunchArgs": "-inprivate"
      }
    ]
    }
  ]
}
```

**Reply by U-C-S** [reply]
haha... Blame me, I forgot to make those newly added fields optional. Thanks for pointing out. Though it's bit hard to deal with changing json schemes. 

Btw, you don't need to worry about editing `LastUpdated` and `Version`. Those are auto generated and currently don't have any uses apart from giving some info to user

**Reply by grentninja** [reply]
It's alright, I appreciate much of your effort for releasing this version! Keep up the good work :)
